### PR TITLE
[ACS-5051] Playwright files extracted

### PR DESCRIPTION
**Pull request by azakrzewski-hy**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Playwright tests can't run from VS IDE.
Playwright tests are present in the same package as Protractor tests
https://alfresco.atlassian.net/browse/ACS-5051

**What is the new behaviour?**
Playwright tests can be run from VS IDE
Playwright tests are separated from Protractor tests

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: e2e-playwright/playwright.config.ts [hunk 1]
```diff
@@ -24,10 +24,10 @@
 
 import { PlaywrightTestConfig, devices } from '@playwright/test';
 
-require('dotenv').config();
+require('dotenv').config({ path: '../.env' });
```
[the remark this answers is not in the document]
Comment (MichalFidor) [reply e2e-playwright/playwright.config.ts:27]: We don't need this path in the bracket. Without it, it still will consider as the `.env` file will be in the root folder.

File: e2e-playwright/storage-state/AdminUserState.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{
```
Comment (MichalFidor) [inline e2e-playwright/storage-state/AdminUserState.json:1]: This file should never be committed. Remove it ASAP!

File: e2e/tsconfig.e2e.json [hunk 3]
```diff
@@ -5,11 +5,10 @@
     "baseUrl": "./",
```
Comment (MichalFidor) [inline e2e/tsconfig.e2e.json:5]: Do we need this file and whole e2e folder?

**Review by MichalFidor — COMMENTED** [review]
As we've spoke with Adam, you don't need to split the page-object, fixtures and other stuff to have it as a shared files. The structure should be like this:

```
e2e-playwright/
├─ page-object/
 │  ├─ components/
 │  ├─ pages/
├─ fixtures/
├─ models/
├─ api/
├─ tests/
 │  ├─ <here you should have only files with tests>
```

Instead of having page-object in shared folder and also inside of the tests folder. You don't need to share anything if you won't use `projects` as it is in monorepository. If you would like to split the tests by groups, you can do that by providing the data in the `playwright.config.ts` under the `projects:[...]`. 

In your use case I don't see any benefits of having it done it the way as it is in this PR.

**Review by DenysVuika — APPROVED** [review]

**Review by akashrathod28 — APPROVED** [review]

**Comment by azakrzewski-hy** [comment]
Closing this PR as it will not be merged as serious refactoring is needed to run playwright tests from VS IDE by using plugins: Playwright Test for VSCode or Nx Console

**Pull request closed without merging**